test/channels/application_cable/connection_test.rb

```diff
@@ -1,4 +1,4 @@
-require "test_helper"
+require 'test_helper'
 
```

Comment:
# Improve on the commit messages
- [  ] Your commit messages aren't as clear as they are expected on this project. In your next round of changes, let's improve on this area. thank you. ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/descriptive_commits.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/)).